### ISSUE TITLE
Suggestion to Disconnect: Always block Content

### ISSUE THREAD

**Issue by ghost** [disconnectme/disconnect#331]
As Disconnect is now, it does not allow blocking Content by default, you have to manually block it in every website you're visiting, it's pretty annoying I say. To improve privacy and user-end happiness allow blocking Content also by default. Maybe a checkbox would be great inside of Diconnect toolbar icon.

**Comment by barraponto** [comment]
yeah, I'd love to see some settings manager, if only to manage this precious setting.

**Comment by truthsword** [comment]
Block by default; whitelist by exception. Yes. Having to use Ghostery to clean up behind Disconnect.

**Comment by damajor** [comment]
Duplicate of #295
